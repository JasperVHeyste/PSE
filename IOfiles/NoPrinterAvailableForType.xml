<?xml version="1.0" encoding="UTF-8"?>
<SYSTEM>
	<COMPENSATION>
		<name>Students 4 Climate</name>
		<compNumber>1234</compNumber>
	</COMPENSATION>
	<DEVICE>
		<name>Office_Printer1</name>
		<emissions>3</emissions>
		<speed>40</speed>
		<cost>10</cost>
		<type>color</type>
	</DEVICE>
	<DEVICE>
		<name>Office_Printer2</name>
		<emissions>3</emissions>
		<speed>40</speed>
		<cost>5</cost>
		<type>bw</type>
	</DEVICE>
	<JOB>
		<jobNumber>1</jobNumber>
		<pageCount>2</pageCount>
		<userName>SergeDemeyer</userName>
		<compNumber>1234</compNumber>
		<type>color</type>
	</JOB>
	<JOB>
		<jobNumber>2</jobNumber>
		<pageCount>3</pageCount>
		<userName>anonymous_user</userName>
		<type>bw</type>
	</JOB>
	<JOB>
		<jobNumber>3</jobNumber>
		<pageCount>3</pageCount>
		<userName>anonymous_user</userName>
		<type>color</type>
	</JOB>
	<JOB>
		<jobNumber>4</jobNumber>
		<pageCount>3</pageCount>
		<userName>anonymous_user</userName>
		<type>color</type>
	</JOB>
	<JOB>
		<jobNumber>5</jobNumber>
		<pageCount>3</pageCount>
		<userName>anonymous_user</userName>
		<type>scan</type>
	</JOB>
</SYSTEM>
