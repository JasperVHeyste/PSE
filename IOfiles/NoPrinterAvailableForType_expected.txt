No printers available for job type: scan
Printer 'Office_Printer1' finished color-printing job:
    Number: 1
    Submitted by 'SergeDemeyer'
    2 pages
Job 1 was made CO2 neutral with the support of Students 4 Climate.
Printer 'Office_Printer2' finished black and white printing job:
    Number: 2
    Submitted by 'anonymous_user'
    3 pages
Printer 'Office_Printer1' finished color-printing job:
    Number: 3
    Submitted by 'anonymous_user'
    3 pages
Printer 'Office_Printer1' finished color-printing job:
    Number: 4
    Submitted by 'anonymous_user'
    3 pages
